CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contributing
+
+Thanks for taking the time to join our community and start contributing!
+
+Please remember to read and observe the [Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+
+This  project accepts contribution via github [pull requests](https://help.github.com/articles/about-pull-requests/). This document outlines the process to help get your contribution accepted. Please also read the [Kubernetes contributor guide](https://github.com/kubernetes/community/blob/master/contributors/guide/README.md).
+
+The [Contributor Guide](https://github.com/kubernetes/community/blob/master/contributors/guide/README.md)
+provides detailed instructions on how to get your ideas and bug fixes seen and accepted.
+
+## Sign the Contributor License Agreement
+We'd love to accept your patches! Before we can accept them you need to sign Cloud Native Computing Foundation (CNCF) [CLA](https://github.com/kubernetes/community/blob/master/CLA.md).
+
+## Reporting an issue
+If you have any problem with the package or any suggestions, please file an [issue](https://github.com/kubernetes-client/java/issues).
+
+## Contributing a Patch
+1. Submit an issue describing your proposed change to the repo.
+2. Fork this repo, develop and test your code changes.
+3. Submit a pull request.
+4. The bot will automatically assigns someone to review your PR. Check the full list of bot commands [here](https://prow.k8s.io/command-help).
+
+### Updating the client
```

Comment:
This should be in the README since it is technical details about how to regenerate the client code, it's not really necessary for most people when they're submitting patches.